get_molecule_xyz/get_molecule_xyz.py

```diff
@@ -7,7 +9,7 @@
 # 
 ####################################################################################################
 
-def cas_to_xyz(xyz_title:str,cas_number:str) -> None:
+def cas_to_xyz(xyz_title:str,cas_number:str) -> str:
```

Comment:
perhaps the functions should use the CAS number as the default title if not title is provided
Since the CAS number is the most essential to this function, it should come first in the variable assignment